portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase

```diff
@@ -273,8 +273,13 @@ definition {
 	}
 
 	@description = "Bug ID: LRAC-8093 | Automation ID: LRAC-11533 | Test Summary: Able to clear property data"
+	@ignore = "true"
 	@priority = 4
 	test ClearPropertyData {
+
+		// AC Refactor: data doesn't appear on time
+		// AC Refactor ticket: LRAC-8994
```

Comment:
Hey @joshitagaki, would this be the correct ticket? The ticket name is not related to this test

Reply:
Oops good catch. Copied the wrong refactor ticket lol